docs/source/technical_design/localinstall.rst

```diff
@@ -84,6 +84,11 @@ In some cases this may not work because Docker does not yet know your user name.
 
 Then OpenKAT is built, including all the parts such as Octopoes and Rocky.
 
+Front end
+*********
+
+Find the frontend of your OpenKAT install at port 8000 or 8443 of your localhost and follow the 'on boarding flow' to test your setup and start using your development setup of OpenKAT.
```

Comment:
we should clarify the protocol here.
http for 8000
https for 8443
Using the wrong protocol to connect will result the browser and server not being happy,  which will not give a useful error message.

Reply:
We should also clarify that using http only works when connecting to localhost due to the security flags on the session and xsrf cookies. Localhost specifically is whitelisted to allow secure cookies over an unsecure connection, whereas connecting to any other IP over http be result in these cookies being disregarded. resulting in XSRF warnings when logging in.

Reply:
done, thnx!